### PR TITLE
Add PoR zone linking support

### PR DESCRIPTION
**Pull request by xackery**
This introduces support for the Prophecy of Ro zones.

This is currently a draft as I'm still testing all the zone points and linking.

- I have a .sql file I'm preparing to fix all zone points as well, and will see about getting that into PEQ's db around same time this PR is completed.
- Feel free to give input on general structure and design. We still haven't made an .editconfig to enforce a standard, once we do I'll convert to that.
- I'm not sure if we want ; on lua, or () on conditionals, I'm happy to follow a standard but seems quests isn't very standard anyways
- Tab 4 ?
- Renamed all zones from "magic numbers" to lua zone consts where applicable of being changed

### CODE REVIEW

File: draniksscar/Elder_Priest_of_Discord.lua [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            e.other:MovePC(Zone.qeynos2, 227, 22, 3, 10)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.rivervale then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.rivervale, -110, -16, 4, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == 55 then
```
Comment (Kinglykrab) [inline draniksscar/Elder_Priest_of_Discord.lua:85]: `55` should be a constant, right?

Comment (xackery) [reply draniksscar/Elder_Priest_of_Discord.lua:85]: Correct. Also seems trust has a tool to turn this into a global script, with his PoR effort

Comment (xackery) [reply draniksscar/Elder_Priest_of_Discord.lua]: Added. This will be in draft for a while, there's a bunch of coords needing updates we're working on

Comment (xackery) [reply draniksscar/Elder_Priest_of_Discord.lua]: Bleh, looks like formatter applied too. I'll reverse it.

**Pull request closed without merging**